src/components/community/IntroCommunity.jsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import styles from './IntroCommunity.module.scss';
```

Comment:
Well I agree with using css modules, but I prefer using tailwindcss.

Reply:
Yes, we can use css modules, and what I mean was if tailwind is possible please use that. :)